src/components/structures/AutoHideScrollbar.js

```diff
@@ -118,12 +118,19 @@ export default class AutoHideScrollbar extends React.Component {
         return this.containerRef.scrollTop;
     }
 
+    onWheel = (e) => {
+        if (this.props.onWheel) {
+            this.props.onWheel(e);
+        }
+    };
+
     render() {
         return (<div
                     ref={this._collectContainerRef}
                     style={this.props.style}
                     className={["mx_AutoHideScrollbar", this.props.className].join(" ")}
                     onScroll={this.props.onScroll}
+                    onWheel={this.onWheel}
```

Comment:
maybe only add the listener if needed?

Reply:
react doesn't seem to care if it is null/undefined, so I've gone ahead and just removed the proxy function.